### DISCUSSION TITLE
Real damage vs decals

### DISCUSSION THREAD

**Discussion by TGessCZ** [tsunamayo/Starship-EVO#4252] [Suggestions]
So I switched from experimental to normal build and the block removal is so much better and satisfying in PVP rather than static ships that look the same just with some stickers on them. 

Is there anyone else who thinks block removal is better than decal damage?

**Comment by Womble-TC** [comment]
I like the currect decal damage and brick pen. I think its preferable asthetically (especially with the smoke effects from broken systems), but it also means you dont end up with weird floating bricks that are still technically attactched to your ship. However, I think some extra damage decals would be nice.

**Comment by pinesh** [comment]
I agree with Womble here, Breaking blocks opens a whole host of bad edge cases with the stretch tech, Things like individual block health would need to be considered or massive swaths of a ship might vanish in a single shot of a low powered weapon.
